www/about/teams.spt

```diff
@@ -129,6 +129,11 @@ title = _("Teams")
     "been created after the most recent payday)."
 ) }}</p>
 
+<p>{{ _(
+    "You can enable or not this option a the very moment you're creating a team. "
+    "If you want to change this option later, you need to contact our support."
```

Comment:
Include a link to email?

Reply:
I thought about that when writing it but I saw there is another "contact" in the "Removing team membership" of the https://liberapay.com/about/teams page, plus there is a "contact" tab at the beginning of the same page, so I ended not putting a link.

Reply:
Right on.